### PR TITLE
tools-kompose-index-pr-2017-09-17

### PR DESCRIPTION
**Pull request by GavinBF**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: docs/tools/kompose/index.md [hunk 1]
```diff
@@ -3,16 +3,21 @@
 approvers:
 - cdrage
 
-title: Kompose Overview
+标题: Kompose 概览
```
Comment (zhangqx2010) [inline docs/tools/kompose/index.md]: title不用翻译

File: docs/tools/kompose/index.md [hunk 2]
```diff
@@ -3,16 +3,21 @@
...
 ---
 
-`kompose` is a tool to help users who are familiar with `docker-compose` move to **Kubernetes**. `kompose` takes a Docker Compose file and translates it into Kubernetes resources.
-
-`kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. Transformation of the Docker Compose format to Kubernetes resources manifest may not be exact, but it helps tremendously when first deploying an application on Kubernetes.
-
-## Use Case
+<!--`kompose` is a tool to help users who are familiar with `docker-compose` move to **Kubernetes**. `kompose` takes a Docker Compose file and translates it into Kubernetes resources.
+-->
+`kompose` 是一个将`docker-compose`迁移到**kubernetes**的工具，`kompose`会把Docker Compose文件翻译成Kubernetes资源文件。
```
Comment (zhangqx2010) [inline docs/tools/kompose/index.md]: `docker-compose` 和 **kubernetes** ，后面的英文也是两边加空格

File: docs/tools/kompose/index.md [hunk 3]
```diff
@@ -3,16 +3,21 @@
...
+<!--`kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. Transformation of the Docker Compose format to Kubernetes resources manifest may not be exact, but it helps tremendously when first deploying an application on Kubernetes.
+-->
+`kompose` 是从本地Docker管理到使用Kubernetes管理你的应用程序的便利工具。 Docker的转换撰写格式到Kubernetes资源清单可能不是精确的，但是当首次在Kubernetes上部署应用程序时，它会起到一定作用。
```
Comment (zhangqx2010) [inline docs/tools/kompose/index.md]: 句号后面不用空格 local Docker development
后面一句读起来不通顺

File: docs/tools/kompose/index.md [hunk 4]
```diff
@@ -3,16 +3,21 @@
...
 
+## 用例说明
+<!--
 If you have a Docker Compose `docker-compose.yml` or a Docker Distributed Application Bundle `docker-compose-bundle.dab` file, you can convert it into Kubernetes deployments and services like this:
+-->
+如果你有一个Docker Compose的`docker-compose.yml`文件或者一个`docker-compose-bundle.dab`文件，可以通过kompose命令将它们生成为Kubernetes的deplyment、service的资源文件，如下所示：
```
Comment (zhangqx2010) [inline docs/tools/kompose/index.md]: Docker Distributed Application Bundle 未翻译

**Review by zhangqx2010 — CHANGES_REQUESTED** [review]
多处需要修改

**Comment by markthink** [comment]
@GavinBF 更新一下吧

**Comment by GavinBF** [comment]
@zhangqx2010 感谢老司机帮助

**Pull request merged**